src/functions.php

```diff
@@ -205,17 +205,28 @@ function resolve_uri($id, $parentScope)
     $parts = Uri\parse($id);
 
     if (!empty($parts['path'])) {
-        // If the path ends in a slash, it shouldn't be replaced but instead appended to.
+        // If the original path ends in a slash, it shouldn't be replaced but instead appended to.
```

Comment:
I _thought_ there was a reason I couldn't use it, but all of the tests seem to pass.

IIRC the issue was just that the spec expected me to keep an extraneous '#' if it existed on the base URI, even if there wasn't a fragment.  It doesn't actually affect schema resolution though.

Anyway, that saves me from maintaining quite a bit of error prone code, so thanks 👍